sections/02_intro.md

```diff
@@ -1,32 +1,33 @@
 ## Introduction
 
-Biology and medicine are rapidly becoming particularly data-intensive with
+Biology and medicine are rapidly becoming data-intensive with
 respect to both research and practice. A recent comparison of genomics with
 social media, online videos and other data-intensive scientific disciplines
 suggested that the field of genomics alone would equal or surpass other fields
 in data generation and analysis within the next decade
 [@doi:10.1371/journal.pbio.1002195]. These data present new opportunities, but
-also new challenges. We expect that algorithms to automatically extract
+also new challenges. The data volume and complexity both indicate that
+automated algorithms will be needed to extract
 meaningful patterns and provide actionable knowledge allowing us to better
 treat, categorize, or study disease, all within data constrained and privacy
-critical environments. will be required.
+critical environments.
 
 Concurrent with this explosive growth in biomedical data, a new class of machine
-learning algorithm was initially popularized in the domain of image analysis.
-Computer scientists are now building many-layered neural networks from
+learning algorithm - artificial neural networks, also known as deep learning -
+is revolutionizing domains from X to Y (I dunno - playing chess? serving ads?).
```

Comment:
Hah! Yes! And finding cats ;) I will leave this in and we can choose exemplar fields.